### ISSUE TITLE
sad loss is very small when adding it to erfnet

### ISSUE THREAD

**Issue by mengzhibin** [cardwing/Codes-for-Lane-Detection#180]
After spatial softmax, each value is close to 0. BSE loss of spatial softmax could be 1e-9, so is there any problem?

**Comment by cardwing** [comment]
What is BSE loss? Besides, do not make the L2 loss divide by H x W.

**Comment by mengzhibin** [comment]
I am sorry, it's MSE. I have already known where is the problem that I have divide by HxW. thanks

**Comment by freshfruit930** [comment]
@mengzhibin  From your post seems that you have add the SAD loss to the ERFnetwork, do you mind to share how you add it? just as the loss function in the enet-label-torch?

Moreover, do you also add the SAD to the stages of the erf network?

**Comment by miao02830** [comment]
> 在空间softmax之后，每个值都接近0。空间softmax的BSE损失可能是1e-9，那么有什么问题吗？

您好，可以分享一下您将sad加入erfnet的代码吗，我的邮箱是931340436@qq.com 十分感谢您
